src/dock_applet.in

```diff
@@ -79,13 +80,6 @@ keyb_shortcuts = ["<Super>1", "<Super>2", "<Super>3", "<Super>4", "<Super>5",
                   "<Super><Alt>1", "<Super><Alt>2", "<Super><Alt>3", "<Super><Alt>4", "<Super><Alt>5",
                   "<Super><Alt>6", "<Super><Alt>7", "<Super><Alt>8", "<Super><Alt>9", "<Super><Alt>0"]
 
-# additional shortcuts for the number pad - definitely don't work on my UK keyboards but included
-# anyway in case they work for other layouts
-keyb_shortcuts1 = ["<Super>KP_1", "<Super>KP_2", "<Super>KP_3", "<Super>KP_4", "<Super>KP_5",
```

Comment:
I could not get these to work at all. I'm wondering if the issue has to do with the modifier masks. I removed them so far due to the complexity of figuring out which application they map to, given the current changes to the code.